courses/management/commands/update_certificate_revision_to_latest.py

```diff
@@ -0,0 +1,46 @@
+from django.core.management.base import BaseCommand, CommandError
+from courses.models import CourseRun, CourseRunCertificate, Program, ProgramCertificate
+from courses.management.utils import update_certificates
+
+
+class Command(BaseCommand):
+    """
+    Change the certificate revision to the latest for the specified course run or program.
+    """
+
+    def add_arguments(self, parser):
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument("--course_run_id", type=int, help="ID of the course run")
+        group.add_argument("--program_id", type=int, help="ID of the program")
```

Comment:
Generally, we name these args as `--course-run-id` and `--program-id`, and then we use them like you used.